[DEVCOMMS-483] Feature/localization section revamp (#1637)
mercadopago/devsite-docs: guides/resources/consult-payment-methods.en.md

# Availability of products and payment methods

The products and payment methods offered by Mercado Pago vary according to each country. See below how to consult all the payment methods that are available and, from that, which will or will not be available by product and by country.

## Consult payment methods

> To consult all available payment methods and obtain a list with the details of each one of them and their properties, perform a GET to the endpoint[/v1/payment_methods](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/reference/payment_methods/_payment_methods/get). You can get more information about the attributes in [Get payment methods](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/reference/payment_methods/_payment_methods/get) in the API references.

Reviewer: Payment methods tiene un solo endpoint, asi que los dos links apuntan al mismo documento. Es necesario poner los dos?

> When consulting the available payment methods, the `response` will indicate the results that will coincide with the country associated with your Mercado Pago account. Check the [payment methods](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/payment_methods) available by country for more information.

Reviewer: el link no me funciona, es para guias independientes?

## Products and payment methods available by country
 

> It may be that some payment methods are not available for some products. Below are detailed which payment methods are not available by product and by country.

Reviewer: ```suggestion
It may be that some payment methods are not available for some products. Payment methods availability by product and by country are detailed below.
```

> It may be that some payment methods are not available for some products. Below are detailed which payment methods are not available by product and by country.

Author: ```suggestion
See below which are the **products available by country** and, from that, which **means of payment will or will not be available by product and by country**.
```

> It may be that some payment methods are not available for some products. Below are detailed which payment methods are not available by product and by country.

Author: ```suggestion
See below which are the **products available by country** and, from that, which **means of payment will or will not be available by product and by country**.
```

 
### Payments:
 
| Product | ARG | BRA | CHL | COL | MEX | PER | URY | VEN |
| :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- |
| Payment link	 | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |
| Checkout Pro | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |
| Mobile Checkout | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | | ✔ |
| Chackout API | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |
 
### Subscriptions:
 
| Product | ARG | BRA | CHL | COL | MEX | PER | URY | VEN |
| :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- |
| Payment link	 | ✔ | ✔ | | | ✔ | | | |
| Checkout Pro | ✔ | ✔ | | | ✔ | | | |
| Checkout API | ✔ | ✔ | | | ✔ | | | |
 

> Based on the previous information, below are detailed which payment methods are not available by country.

Reviewer: ```suggestion
Based on the previous information, below we detail which payment methods are available/unavailable by country.
```
